### PR TITLE
Update the_big_picture.rst

### PR DESCRIPTION
**Pull request by WDI-Json**
Walkthrough first page setting up with PHP 8.3 and fresh installation of Symfony 7. I received errors with a clean setup while going through this and fixed them using my changes.

### CODE REVIEW

**Comment by WDI-Json** [comment]
The changes aren't that much, but I needed these in order to make it work somehow :/

**Comment by OskarStark** [comment]
Symfony 7 requires PHP 8.2

**Comment by WDI-Json** [comment]
@OskarStark I've changed to `8.2.0` as stated in https://symfony.com/releases/7.0

File: quick_tour/the_big_picture.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ safe & easy!) and offers long-term support.
 Downloading Symfony
 -------------------
 
-First, make sure you've installed `Composer`_ and have PHP 8.1 or higher.
+First, make sure you've installed `Composer`_ and have PHP 8.2.0 or higher.
```
Comment (xabbuh) [inline quick_tour/the_big_picture.rst]:
```suggestion
First, make sure you've installed `Composer`_ and have PHP 8.2 or higher.
```

**Comment by xabbuh** [comment]
I think we even need to do a bit more as the skeleton application looks a bit different than documented since Symfony 6 (see #19834 for my proposal).

**Comment by javiereguiluz** [comment]
Let's close this proposal in favor of #19834, which was already merged a while ago.

Even if this wasn't merged, thanks for creating this PR because it helped us improve the related docs. Thanks!

**Pull request closed without merging**